models/marts/fct_order_items.sql

```diff
@@ -0,0 +1,59 @@
+with order_items as  (
+    select * from {{ ref('stg_order_items' )}}
+),
+
+orders as  (
+    select * from {{ ref('stg_orders' )}}
+),
+
+order_payments as  (
+    select * from {{ ref('stg_order_payments' )}}
+),
+
+reviews as (
+    select * from {{ ref('stg_order_reviews') }}
+),
+
+final as (
+
+    select
+    -- primary key
+        order_items.order_item_sk,
+    -- foreign keys
+        order_items.order_id,
+        order_items.product_id,
+        order_items.seller_id,
+        orders.customer_id,
+    -- timestamps
+        orders.order_purchase_timestamp as ordered_at,
+        orders.order_approved_at as approved_at,
+        orders.order_delivered_carrier_date as delivered_carrier_at,
+        orders.order_delivered_customer_date as delivered_customer_at,
+        orders.order_estimated_delivery_date as estimated_delivery_at,
+    -- dimensions
+        order_items.price,
+        order_items.freight_value,
+        case when order_payments.payment_type is null then 'returned'
+            else orders.order_status end as order_status,
+        coalesce(order_payments.payment_type, 'returned') as payment_type,
+        coalesce(order_payments.payment_installments, 0) as payment_installments,
+        coalesce(order_payments.payment_value, 0) as payment_amount,
+        reviews.review_score as order_review_score,
```

Comment:
this type of renaming should also happen in the stage model

Reply:
noted, removed renaming for this specific use case .. upon revision it does not seem necessary